docs/reference/components.rst

```diff
@@ -101,3 +101,15 @@ Item metadata components
 
 .. autoclass:: zyte_common_items.RealEstateMetadata(**kwargs)
    :members: dateDownloaded, probability
+
+.. autoclass:: zyte_common_items.JobPostingMetadata(**kwargs)
+   :members: dateDownloaded, probability
+
+.. autoclass:: zyte_common_items.BaseSalary(**kwargs)
+   :members: raw, valueMin, valueMax, rateType, currency, currencyRaw
+
+.. autoclass:: zyte_common_items.HiringOrganization(**kwargs)
+   :members: name, nameRaw, id
+
+.. autoclass:: zyte_common_items.JobLocation(**kwargs)
+   :members: raw
```

Comment:
The metadata class should stay in this section, but properly sorted, alphabetically.

The rest of components should be on the earlier section.